index.html

```diff
@@ -0,0 +1,496 @@
+                <i class="fas fa-times dismiss"></i>
+                <img class="hcard-image" src="https://n2.sdlcdn.com/imgs/g/6/x/HRX-MJ10242A-Sneakers-Black-Casual-SDL239404586-1-c3fa2.jpeg" alt="image">
+                <div class="card-info">
+                    <div class="card-description">
+                        <h2>Hrx by Hrithik Roshan</h2>
+                        <p>Men Grey Rage-2 Running Shoe</p>
+                        <span>Sold by: Omnitech Retail</span>
+                        <p>Designed to provide natural support to your feet and ankle.Constructed with knitted fabric.Cushioned collar.</p>
+                    </div>   
+                    <div class="card-button-option">
+                            <button class="btn btn-primary"><i class="fas fa-shopping-cart"></i> Add to Cart</button>
+                            <button type="button" class="btn btn-outline">Move to Wishlist</button>
+                    </div>
+                </div>   
+            
+        </div>
+        <!--- Cards with text overlay-->
+        <div class="horizontal-card card-dismiss">
+            <i class="fas fa-times dismiss"></i>
+            <h4 class="img-text-overlay">New</h4>
+            <img class="hcard-image" src="https://n2.sdlcdn.com/imgs/g/6/x/HRX-MJ10242A-Sneakers-Black-Casual-SDL239404586-1-c3fa2.jpeg" alt="image">
+            <div class="card-info">
+                <div class="card-description">
+                    <h2>Hrx by Hrithik Roshan</h2>
+                    <p>Men Grey Rage-2 Running Shoe</p>
+                    <span>Sold by: Omnitech Retail</span>
+                    <p>Designed to provide natural support to your feet and ankle.Constructed with knitted fabric.Cushioned collar.</p>
+                </div>   
+                <div class="card-button-option">
+                        <button class="btn btn-primary"><i class="fas fa-shopping-cart"></i> Add to Cart</button>
+                        <button type="button" class="btn btn-outline">Move to Wishlist</button>
+                </div> 
+            </div>   
+        
+        </div>
+    </div>
+
+    <!--VERTICAL CARD-->
+    <div class="information">
+        <header>Vertical Cards</header>
+        <h3>Examples of different Vertical cards are:</h3>
+    </div>
+
+
+    <div class="center">
+        <div class="vertical-card">
+            <img class="card-image" src="https://n2.sdlcdn.com/imgs/g/6/x/HRX-MJ10242A-Sneakers-Black-Casual-SDL239404586-1-c3fa2.jpeg" alt="image">
+            <div class="card-info">
+                <div class="card-description">
+                    <h2>Hrx by Hrithik Roshan</h2>
+                    <p>Men Grey Rage-2 Running Shoe</p>
+                    <span>Sold by: Omnitech Retail</span>
+                    <p>Designed to provide natural support to your feet and ankle.Constructed with knitted fabric.Cushioned collar.</p>
+                </div>   
+                <div class="card-button-option">
```

Comment:
Suggestion : a little margin on top of the button div would make the ui of cards more breatheable

Reply:
Suggestion : There is some blank space at the right side of the vertical cards. You can remove those